topology/Niels Bohr Institute/NBI/NBI_T3.yaml

```
@@ -0,0 +1,57 @@
+Production: true
+SupportCenter: Self Supported
+
+GroupDescription: IceCube Tier 3 located at the Niels Bohr Institute
+GroupID: 509
+
+Resources:
+  NBI_T3_ARC_CE:
```

Let's use the factory convention for the resource name (`<VO>_<COUNTRY CODE>_...`). I don't believe this is a hard requirement but this is how we've been registering the hosted CE resources.

```suggestion
  IceCube_DK_NBI_T3
```

---

We should have CE in there though.

---

IceCube_DK_NBI_T3_CE?

---

Good point, I'll update the suggestion